.gitignore

```diff
@@ -3,6 +3,9 @@
 target
 debug
 
+# generated by build script, used for testing
+_artefacts
```

Comment:
I think just `artifacts` would be fine, no need `_`

Reply:
Do we want to include the initial version of `axelar_gateway_new.wasm` and `axelar_gateway_old.wasm` for testing? Otherwise for someone cloned the repo, running `build.rs` becomes mandatory for `cargo test` to pass, which may be unclear. Alternatively, we could update the `README.md` to have clear instructions